source/02-qubits.Rmd

```diff
@@ -418,7 +418,7 @@ $$
     1+i & 1-i
   \\1-i & 1+i
   \end{bmatrix}
-  = \frac{1}{2}
+  = \frac{1}{\sqrt{2}}
```

Comment:
I am pretty sure that the correct factor is actually indeed 1/2 (in fact, somebody submitted a correction saying exactly this about 2 weeks ago!)

Reply:
(yes, this was https://github.com/thosgood/qubit.guide/commit/fe8418cd63da2dca214243ebde2acba861503b33 , and I believe that I double checked it at the time!)

Reply:
So I think that actually the $\frac{1}{\sqrt(2)}$ is needed, as $e^{i\frac{\pi}{4}} = \frac{1}{\sqrt(2)} + i\frac{1}{\sqrt(2)}$, leaving only a factor of $\frac{1}{\sqrt(2)}$ missing, as opposed to $\frac{1}{2}$. I also submitted the previous correction for this! (I was reading my corrected section with pride, and noticed this factor!)

Reply:
oh sorry, I didn't recognise your name for a moment there! I see what you mean, in which case I think this also needs fixing on lines 489, 501, and 509, right? (i.e. just below Figure 2.2)

Reply:
Yes, you're right! I'll change those now on the PR.

Reply:
Okay, should all be finalised now! Let me know if there are any other changes you spot :)